### PR TITLE
Apply reference review

### PR DESCRIPTION
**Pull request by Jesus89**
Closes https://github.com/CartoDB/cartoframes/issues/1299

### CODE REVIEW

File: cartoframes/data/observatory/catalog/catalog.py [hunk 1]
```diff
@@ -41,13 +41,13 @@ class Catalog(object):
         Let's say you explore a `dataset` with demographic data for the whole US at the Census tract level.
         The variables give you information about the actual columns you have available, such as: total_population,
         total_males, etc.
-        On the other hand, you use `Variable` instances or lists of :py:attr:`Variable.id` or :py:attr:`Dataset.slug`
-        to enrich your own data.
+        On the other hand, you can use `Variable` instances or lists of :py:attr:`Variable.id` or
+        :py:attr:`Dataset.slug` to enrich your own data.
```
Comment (cmongut) [inline cartoframes/data/observatory/catalog/catalog.py]: Is this right? It should be Variable.slug, isn't it?

Comment (Jesus89) [reply cartoframes/data/observatory/catalog/catalog.py]: This what we had before. @oleurud could you check that this is right, or should we change it to Variable.slug?

Comment (oleurud) [reply cartoframes/data/observatory/catalog/catalog.py]: Yes, it should `Variable.slug`. Also, I would say:
```
you can use lists of `Variable` instances, :py:attr:`Variable.id` or :py:attr:`Variable.slug` to enrich your own data.
```

Comment (Jesus89) [reply cartoframes/data/observatory/catalog/catalog.py]: OK. Thanks!

Comment (Jesus89) [reply cartoframes/data/observatory/catalog/catalog.py]: Changed

**Pull request merged**